CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
-## [Unreleased]
+## [Unreleased 0.2.0]
 ### Changed
 - Fixed typos in logback warning messages emitted by `com.github.danielwegener.logback.kafka.KafkaAppenderConfig` (#28)
-- Switched default delivery strategy to `com.github.danielwegener.logback.kafka.delivery.AsynchronousDeliveryStrategy` as it is the [more sensible default](https://github.com/danielwegener/logback-kafka-appender/pull/32). 
+- Switched default delivery strategy to `com.github.danielwegener.logback.kafka.delivery.AsynchronousDeliveryStrategy` as it is the [more sensible default](https://github.com/danielwegener/logback-kafka-appender/pull/32).
+- Make artifact osgi compatible (#40)
+- Upgrade to logback 1.2.3. Dropped `KafkaMessageEncoder` infrastructure in favor of `ch.qos.logback.core.encoder.Encoder` (#51).
+- Upgrade to kafka 0.10.0. Add use the actual log event timestamp to the kafka record.
```

Comment:
Why not use Kafka 0.11?

Reply:
Backward compatibility. You can use a kafka  0.10.0 producer against a 0.11.0 broker. This does (probably) not work the other way around. I also do not see any producer feature in 0.11.0 that we really can benefit from (except you have a good use-case for headers).

Reply:
Since Kafka 0.10.2, there is a protocol version negotiation in the protocol ( https://cwiki.apache.org/confluence/display/KAFKA/KIP-35+-+Retrieving+protocol+version ), this allows the client to be both backward and forward compatible ( https://cwiki.apache.org/confluence/display/KAFKA/KIP-97%3A+Improved+Kafka+Client+RPC+Compatibility+Policy )

Reply:
Good to know, I have missed that one :+1: So IMO we can upgrade to the 0.11 producer, but I'd still run the integration tests against an embedded 0.10.x broker.

Reply:
I think we'll better directly target kafka 1.0 then :)